test-case/manual/check-simple-probe-capture.md

```diff
@@ -0,0 +1,77 @@
+# Probes test
+Simple probes functionality test.
+
+# Preconditions
+1. Linux kernel should have these config options enabled:
+   CONFIG_SND_SOC_SOF_DEBUG_PROBES=y
+   CONFIG_SND_SOC_SOF_HDA_PROBES=y
+
+2. Firmware should have these config options enabled:
+   CONFIG_PROBE=y
+   CONFIG_PROBE_POINTS_MAX=16
+   CONFIG_PROBE_DMA_MAX=4
+
+2. Enable the module, by creating file /etc/modprobe.d/sof-probes.conf, with
+   following line as content:
+   options snd_sof_probes enable=1
+
+3. Add following line to /etc/modprobe.d/sof.conf:
```

Comment:
to make the sound card for the probes to be locked at card3

Reply:
Should be now fixed.